packages/arb-avm-cpp/data_storage/src/messageentry.cpp

```diff
@@ -37,33 +37,44 @@ MessageEntry deserializeMessageEntry(
     std::vector<const unsigned char> entry_vector) {
```

Comment:
Looks like this is being passed by value

Reply:
fixed